test/integration/toolchains/gnu/test_makedeps.py

```diff
@@ -411,6 +411,7 @@ def package_info(self):
     assert "CONAN_NAME_OTHER" not in make_content
 
 
+@pytest.mark.skipif(platform.system() == "Windows", reason="Test uses Unix-style paths not compatible with Windows filesystem.")
```

Comment:
This is unexpected, at least some ``#FIXME`` (as it is not an xfail), better investigate a bit more?